src/components/Navigation.js

```
@@ -0,0 +1,638 @@
+          e.preventDefault();
+          this.focusNextItem();
+        }
+        break;
+        
+      case 'ArrowUp':
+        e.preventDefault();
+        this.focusPreviousItem();
+        break;
+        
+      case 'Tab':
+        if (e.shiftKey) {
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)